src/scss/_uswds-theme.scss

```diff
@@ -0,0 +1,214 @@
+@use 'sass:map';
+
+$site-palette: (
+  // Blue
+  'site-darkest-blue': #0c213a,
+  'site-darker-blue': #112e51,
+  'site-dark-blue': #205493,
+  'site-blue': #0071bb,
+  'site-light-blue': #cedced,
+  'site-lighter-blue': #ebf3fa,
+  'site-lightest-blue': #f2f9ff,
+  // Teal
+  'site-darkest-teal': #006175,
+  'site-darker-teal': #007187,
+  'site-dark-teal': #008099,
+  'site-teal': #00a2c4,
+  'site-light-teal': #7bbfd6,
+  'site-lighter-teal': #b7dde9,
+  'site-lightest-teal': #ecfcff,
+  // Red
+  'site-darkest-red': #680617,
+  'site-darker-red': #8b0a21,
+  'site-dark-red': #ab142f,
+  'site-red': #e21c3d,
+  'site-light-red': #ea7b85,
+  'site-lighter-red': #f3b8be,
+  'site-lightest-red': #fff0f3,
+  // Grey
+  'site-darkest-grey': #1b1b1b,
+  'site-darker-grey': #454545,
+  'site-dark-grey': #5c5c5c,
+  'site-grey': #767676,
+  'site-light-grey': #adadad,
+  'site-lighter-grey': #dedede,
+  'site-lightest-grey': #f0f0f0,
+  'site-ink': #454545,
+  'site-disabled-grey': #767676,
+  // Yellow
+  'site-darkest-yellow': #504332,
+  'site-darker-yellow': #847122,
+  'site-dark-yellow': #bba212,
+  'site-yellow': #f5d600,
+  'site-light-yellow': #f9e450,
+  'site-lighter-yellow': #fcf197,
+  'site-lightest-yellow': #fffdd7,
+  // Green
+  'site-darkest-green': #094316,
+  'site-darker-green': #0e581e,
+  'site-dark-green': #136e26,
+  'site-green': #18852e,
+  'site-light-green': #60ae70,
+  'site-lighter-green': #a6d5af,
+  'site-lightest-green': #ebfcef,
+  // White
+  'site-white': #ffffff
+);
+
+@use 'uswds-core' with (
+  // ----------------------------------------
+  // Colors
+  // ----------------------------------------
```

Comment:
I didn't like having to collapse all the settings here, since we have so many of them that we configure. I need to check syntax support, but I'd wonder if it'd be possible to at least do something like...

```scss
// src/scss/uswds-theme/_color.scss
$settings: (
  $theme-color-base-family: 'grey',
  // ...
);
```

```scss
// src/scss/uswds-theme/_index.scss
@use 'sass:map';
@use 'color';

@use 'uswds-core' with map.merge(color.$settings /* , ... */ );
```

Reply:
I tested this and it did not work. While the `with` parameter looks like it's provided as a map value, that value must be given using the map syntax, and not as a map variable / result of a map operation like `map.merge`.